app/bundles/EmailBundle/Tests/Model/EmailModelFunctionalTest.php

```diff
@@ -79,13 +81,13 @@ private function createSegment(): LeadList
     /**
      * @param Lead[] $contacts
      */
-    private function addContactsToSegment(array $contacts, LeadList $segment): void
+    private function addContactsToSegment(array $contacts, LeadList $segment, ?\DateTime $dateAdded = null): void
```

Comment:
Why this was changed? I don't see this method being called in this PR. Also the `useCleanupRollback` was changed but why?

Reply:
I suppose we want to add more test for setContinueSending true/false  and in this case we check date added to segment